build.rs

```diff
@@ -1,3 +1,12 @@
+#[cfg(not(feature = "build"))]
+fn main() {
+    if cfg!(feature = "ogg") {
+        println!("cargo:rustc-link-lib=ogg");
```

Comment:
Is this really necessary?
It somehow contradicts the change in `tests/ogg_support.rs` doesn't it?

Reply:
Or is this necessary for static linking (I tried only dynamic linking)?

Reply:
To answer my own question:

> Or is this necessary for static linking?

No. I just tried it by disabling my local `libFLAC.so`, leaving only `libFLAC.a`, which still gave me OGG support.